test/jasmine/tests/select_test.js

```diff
@@ -703,4 +703,45 @@ describe('Test select box and lasso per trace:', function() {
         .then(done);
     }, LONG_TIMEOUT_INTERVAL);
 
+    it('should work on choropleth traces', function(done) {
+        gd = createGraphDiv();
```

Comment:
`beforeEach`?

Reply:
done in https://github.com/plotly/plotly.js/pull/2030/commits/a3df09ebea3552210125b1de41b8afb103db125c